Tests/HyloTests/ParserTests.swift

```diff
@@ -1648,6 +1648,196 @@ final class ParserTests: XCTestCase {
     XCTAssertNotNil(try apply(Parser.exprStmt, on: input))
   }
 
+  func testSimpleConditionalControl() throws {
+    let input: SourceFile = "#if os(macOs) foo() #endif"
+    let (stmtID, ast) = try apply(Parser.stmt, on: input)
+    let stmt = try XCTUnwrap(ast[stmtID] as? ConditionalCompilationStmt)
+    XCTAssertEqual(stmt.condition, .os("macOs"))
+    XCTAssertEqual(stmt.stmts.count, 1)
+    XCTAssertEqual(stmt.fallback.count, 0)
+  }
```

Comment:
```suggestion
  }

```

We add one new line between function declarations.